README.md

```diff
@@ -15,27 +15,27 @@ These tests rely on resources and utility modules that have been originally deve
 ### Initial preparation
 
 1. Install frostfs-cli
-    - `git clone git@github.com:nspcc-dev/frostfs-node.git`
+    - `git clone git@github.com:TrueCloudLab/frostfs-node.git`
     - `cd frostfs-node`
     - `make`
     - `sudo cp bin/frostfs-cli /usr/local/bin/frostfs-cli`
 
 2. Install frostfs-authmate
-    - `git clone git@github.com:nspcc-dev/frostfs-s3-gw.git`
+    - `git clone git@github.com:TrueCloudLab/frostfs-s3-gw.git`
     - `cd frostfs-s3-gw`
     - `make`
-    - `sudo cp bin/frostfs-authmate /usr/local/bin/frostfs-authmate`
+    - `sudo cp bin/frostfs-s3-authmate /usr/local/bin/frostfs-authmate`
```

Comment:
It's called everywhere Frostfs S3 Authmate, but as a cli in tests it is referred to as `frostfs-authmate`. We may change it everywhere to `frostfs-s3-authmate`.